compute_tools/src/monitor.rs

```diff
@@ -102,12 +89,110 @@ fn watch_compute_activity(compute: &ComputeNode) {
     }
 }
 
+// Hang on condition variable waiting until the compute status is `Running`.
+fn wait_for_postgres_start(compute: &ComputeNode) {
+    let mut state = compute.state.lock().unwrap();
+    while state.status != ComputeStatus::Running {
+        info!("compute is not running, waiting before monitoring activity");
+        state = compute.state_changed.wait(state).unwrap();
+
+        if state.status == ComputeStatus::Running {
+            break;
+        }
+    }
+}
+
+// Figure out the total active time across all non-system databases.
+// It can return `0.0`, which means no user databases exist.
+fn get_databases_active_time(cli: &mut Client) -> anyhow::Result<f64> {
+    let active_time = cli.query_one(
+        "SELECT coalesce(sum(active_time), 0.0) AS total_active_time
+        FROM pg_stat_database
+        WHERE datname NOT IN (
+                'postgres',
```

Comment:
why exclude `postgres` database here?
users can connect to it with psql (and probably with other drivers too)

I believe some tools may use it as a default dbname.

Reply:
> why exclude postgres database here?

Because we use it as well for monitoring purpose, so every query from postgres_exporter, compute_ctl, pgbouncer, will bump active_time on postgres db

Reply:
>Because we use it as well for monitoring purpose, so every query from postgres_exporter, compute_ctl, pgbouncer, will bump active_time on postgres db

would be nice to add a comment about this